### PR TITLE
feat(lang): add parent dependencies

### PR DESCRIPTION
**Pull request by AndreyLevchenko**

### CODE REVIEW

File: analyzer/language/analyze.go [hunk 1]
```diff
@@ -10,10 +10,11 @@ import (
 	godeptypes "github.com/aquasecurity/go-dep-parser/pkg/types"
 )
 
-type Parser func(r io.Reader) ([]godeptypes.Library, error)
+type Parser func(r io.Reader) ([]godeptypes.Library, []godeptypes.Dependency, error)
```
[the remark this answers is not in the document]
Comment (AndreyLevchenko) [reply analyzer/language/analyze.go:13]: there is no logic in fanal actually. It's same with `parse`.

[the remark this answers is not in the document]
Comment (AndreyLevchenko) [reply analyzer/language/analyze.go:13]: I like it, but we will need introduce type implementation in `go-dep-parser`. Right now it's just set of functions.
but it's something to think about

[the remark this answers is not in the document]
Comment (knqyf263) [reply analyzer/language/analyze.go:13]: We already introduced it to Maven.
https://github.com/aquasecurity/go-dep-parser/blob/main/pkg/java/pom/parse.go#L54-L80

[the remark this answers is not in the document]
Comment (knqyf263) [reply analyzer/language/analyze.go:13]: @AndreyLevchenko How hard is it? If it is not so hard, I think we should define the interface.

**Review by knqyf263 — APPROVED** [review]

**Pull request merged**